### PR TITLE
Add Account API endpoint [ch28502]

### PR DESCRIPTION
**Pull request by MariaBraganca**
Account details such as name, currency, time zone and week start on are now available via API.
The ruby library was updated to include this new endpoint.

```Ruby
ChartMogul::Acount.retrieve
```
```Bash
#<ChartMogul::Account:0x007fb4993f25a0 
@name="Example Test Company",
@currency="EUR", 
@time_zone="Europe/Berlin", 
@week_start_on="sunday"
>
```
Find more information [here](https://www.notion.so/chartmogul/WIP-App-Make-account-details-available-via-API-calls-379e0e91d8d94a37ae04ffdfcf57c8c7) on notion.

Link to [Clubhouse Story](https://app.clubhouse.io/chartmogul/story/28502/update-libraries-for-account-api-endpoints).

### CODE REVIEW

File: lib/chartmogul/account.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+module ChartMogul
+  class Account < APIResource
+    set_resource_name 'Account'
+    set_resource_path '/v1/account'
+
+    readonly_attr :name
+    readonly_attr :currency
+    readonly_attr :time_zone
+    readonly_attr :week_start_on
```
Comment (pkopac) [inline lib/chartmogul/account.rb:11]:
```suggestion
    readonly_attr :week_starts_on
```
?

File: spec/chartmogul/account_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ChartMogul::Account, uses_api: true do
+  describe 'API interactions', vcr: true, record: :all do
+    let(:account) { ChartMogul::Account.retrieve }
+
+    it 'returns the name of current account', uses_api: true do
```
Comment (pkopac) [inline spec/chartmogul/account_spec.rb:9]: Can you please record just one VCR, ie. make it use the same one?

Comment (MariaBraganca) [reply spec/chartmogul/account_spec.rb:13]: Done!

**Review by pkopac — APPROVED** [review]

**Review by ikanade — APPROVED** [review]

**Review by tommycarney — APPROVED** [review]

**Review by polis80cy — APPROVED** [review]

**Pull request merged**